tests/functionality/cgmSimulatorScreen.tests.js

```diff
@@ -1,4 +1,4 @@
-var cgmSimulatorScreen = (test) => {
+module.exports = cgmSimulatorScreen = (test) => {
```

Comment:
[nit] Same here. Is `cgmSimulatorScreen = ` necessary? This would apply in all similar cases (and I'll stop commenting on them now).

Reply:
good point - have cleaned them all up